### PR TITLE
Only allow verification for users with a completed profile

### PR DESCRIPTION
**Pull request by verdefred**
## Summary
We're having some verification requests for users with no profile. We want to have a high standard for our profiles and people that can mint our NFTs so we're adding a validation to check if the user has the profile completed before allowing to do the verification process

## Notes

image
image
image
image

### CODE REVIEW

File: app/models/talent.rb [hunk 1]
```diff
@@ -78,6 +78,10 @@ def profile_picture_url
     profile_picture(:default)&.url || super
   end
 
+  def profile_completed?
```
Comment (RubenSousaDinis) [inline app/models/talent.rb:81]: I think this method belongs to the user model. We already have a similar one there: https://github.com/talentprotocol/web-dapp/blob/bd1e4514387b2a0d2440f49399d9ed9c9bd22904/app/models/user.rb#L123-L125

**Review by RubenSousaDinis — COMMENTED** [review]
Just one comment

File: app/models/talent.rb (hunk 1 above)
Comment (verdefred) [reply app/models/talent.rb:81]: Good catch, I got distracted with the scope profile_complete in here. I'll make the change

**Review by RubenSousaDinis — APPROVED** [review]
Well done!

File: app/packs/src/components/profile/Overview.jsx [hunk 2]
```diff
@@ -553,16 +553,34 @@ const Overview = ({
...
-                              onClick={() => verifyTalent()}
-                            />
-                          )}
+                        {!talent.verified && (
+                          <Button
+                            className="mr-2"
+                            type="primary-default"
+                            onClick={() => verifyTalent()}
+                            disabled={
+                              !talent.profileCompleted || talent.withPersonaId
```
Comment (RubenSousaDinis) [inline app/packs/src/components/profile/Overview.jsx:562]: should this be `user.profileCompleted`?

**Review by RubenSousaDinis — APPROVED** [review]

**Pull request merged**